openmicroscopy/openmicroscopy
# Downgrade insight

In this PR, review the way we read entries from the catalog
To test:
 * Select an image. 
 * Select to export it as ome-tiff
 * Check that the menu with older versions is displayed
 * Select an older version and click export.
 * Re-import the generated ome-tiff

This should now work on mac windows and linux
Check if the option is also available when using webstart
